app/templates/partials/questions/repeatinganswer.html

```diff
@@ -27,7 +29,9 @@ <h3 class="neptune">{{ _('Person ') }}<span class="js-household-loopindex">{{ lo
         <fieldset class="question__fieldset">
           <legend class="question__legend u-vh">{{ _('Person ') }} {{ loop.index }}</legend>
           {% for answer in composite_answer %}
-            {% include theme('partials/answer.html') %}
+            {% with render_guidance = False %}
```

Comment:
The flag is used to not display the guidance in the shared answer template for repeating answers. I think this is good enough to go in as we need to review all of the template handling.